crates/control/src/led_status.rs

```diff
@@ -207,30 +207,40 @@ impl LedStatus {
         blink_state: bool,
         current_maximum_temperature: f32,
     ) -> Ear {
-        if filter_whistle_detected {
-            return Ear::full_ears(1.0);
-        }
+        let temperature_level_one = 76.0;
+        let temperature_level_two = 80.0;
+        let temperature_level_three = 90.0;
```

Comment:
Make these constants, move these nearer to the uses (e.g. above line 227).

Why did you define constants for the temperature percentages but not for the LED percentages?

Reply:
Done. 

Because the temperature levels are globally valid, the percentages for the ears are just an implementation detail.

Reply:
Since the temperature levels are also implementation details of the LED percentage calculation, I would still proposed to move them nearer to the uses (above line 227).

And the value that you are storing in the variable is not a `discrete_temperature` aka a temperature but a percentage that is used in the LEDs. This name can be improved.

Reply:
Ah sorry, misread the line where you wanted the constants to be defined.
Actually, these values are not an implementation detail of the LED percentage calculation, they are defined by the NAO.

Reply:
Yeah, but they are only relevant in this node in the `else` block, so they are an implementation detail of this `else` block and the constants should not leak into the outer scope.

Thanks for fixing it. Resolving.

Reply:
Where do you suggest to define these constants globally?